.github/workflows/pr_cherrypick_prestep.yml

```diff
@@ -0,0 +1,123 @@
+        pr_id=$(echo $GITHUB_CONTEXT | jq ".event.number" | sed 's/"//g')
+        pr_url=$(echo $GITHUB_CONTEXT | jq ".event.pull_request._links.html.href" | sed 's/"//g')
+        repository=$(echo $GITHUB_CONTEXT | jq ".repository" | sed 's/"//g')
+        labels_count=$(echo $GITHUB_CONTEXT | jq ".event.pull_request.labels | length")
+        labels_json=$(echo $GITHUB_CONTEXT | jq ".event.pull_request.labels")
+        echo =============================
+        echo SHA:           $sha
+        echo PRID:          $pr_id
+        echo pr_url:        $pr_url
+        echo repository:    $repository
+        echo labels_count:  $labels_count
+        echo labels_json：   $labels_json
+        echo ${TOKEN} | gh auth login --with-token
+        echo =============================
+        if [[ $(echo $GITHUB_CONTEXT | jq ".event.action" | sed 's/"//g') == "labeled" ]] && \
+           [[ $(echo $GITHUB_CONTEXT | jq ".event.label.name" | sed 's/"//g') != "Approved for ${{ matrix.branch }} Branch" ]];then
+          echo "Newly added label didn't match."
+          exit 0
+        fi
+        create_pr=''
+        for (( i=0; i<$labels_count; i++ ))
+        do
+          label=$(echo $labels_json | jq ".[$i].name" | sed 's/"//g')
+          echo label ${i}: $label
+          if echo $label | grep -oE "Approved for ${{ matrix.branch }} Branch";then
+            create_pr=1
+          fi
+          if echo $label | grep -oE "Created PR to ${{ matrix.branch }} Branch";then
+            echo "already has tag: Created PR to ${{ matrix.branch }} Branch, exit"
+            exit 0
+          fi
+          if echo $label | grep -oE "Included in ${{ matrix.branch }} Branch";then
+            echo "already has tag: Included in ${{ matrix.branch }} Branch, exit"
+            exit 0
+          fi
+        done
+        if [[ "$create_pr" != "1" ]];then
+          echo "Didn't find 'Approved for ${{ matrix.branch }} Branch' tag."
+          exit 0
+        fi
+
+        # Begin to cherry-pick PR
+        git checkout -b ${{ matrix.branch }} --track origin/${{ matrix.branch }}
+        git checkout .
+        git clean -xdff
+
+        git config user.name mssonicbld
+        git config user.email sonicbuildadmin@microsoft.com
+
+        git config credential.https://github.com.username mssonicbld
+        git remote add mssonicbld https://mssonicbld:${TOKEN}@github.com/mssonicbld/sonic-buildimage
+        git fetch mssonicbld
+        git remote -vv
+
+        if ! git cherry-pick $sha;then
+          echo 'cherry-pick failed.'
+          # Add label
+          gh pr edit $pr_url --add-label "Cherry Pick Conflict_${{ matrix.branch }}"
+          echo 'Add label "Cherry Pick Conflict_${{ matrix.branch }}" success'
+          author=$(gh pr view $pr_url --json author | jq '.author.login' | sed 's/"//g')
+          gh pr comment $pr_url --body "@${author} PR conflicts with ${{ matrix.branch }} branch"
```

Comment:
How about creating an issue to the author? Better to summarize the conflict branches.

Reply:
author maybe don't care of issues. But author will take care of their PRs.
And they will receive email when comment on their PRs.